ui_framework/doc_site/src/views/tool_bar/tool_bar_search_only.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+} from '../../../../components';
+
+export const ToolBarSearchOnly = () => (
+  <KuiToolBar className="kuiToolBar--searchOnly">
```

Comment:
I see the benefit of this being if we ever want to change the css name, it's not a large refactor, just change it in one place, and it's also more readable - isSearchOnly vs the css name.

On the other hand, if any time we use a component that requires some custom css based on context, I could see that growing out of hand.  isSearchOnly, isPagerOnly, isOnlySearchAndPager, isPagerAndText, etc, etc, etc.  Where do we draw the line?  We'd also currently be supporting a situation that we only ever use in our docs, not actually in Kibana (I think every table also has a pager so it wouldn't be used in practice).  We support the html attribute className so people can tweak the basics depending on their unique use case, and this I see being a special case because it's only used in documentation, not in kibana.  Maybe that's a problem and we shouldn't be showcasing an example that doesn't actually exist in our code currently.

On yet another hand, if we do think a toolbar with a single search bar will be used frequently then maybe we should just make it a separate component instead of passing in a prop?  (I don't think that's what you meant by `ToolBarSearch` comment above, which I took to mean, a search section in a tool bar that can encompass other things like a select, but maybe I'm wrong).

How would you make the decision for when to create a component and when to use css based on a prop?  The benefit of a component is that it encapsulates the logic more. I want a toolbar with only a search box, and I as an engineer, don't have to know that I have to pass a prop based on the contents.  On the other hand, I have to know this special component exists and I should use it, instead of creating my own out of the tools given.